DXRando/DeusEx/Classes/FlagToggleTrigger.uc

```diff
@@ -0,0 +1,39 @@
+class FlagToggleTrigger extends Trigger;
+
+var name flagName;
+var int flagExpiration;
+
+static function FlagToggleTrigger CreateFTT(Actor a, name event, name flagName, optional int flagExpiration)
```

Comment:
I would probably label this as the tag, not the event, since you're passing it in to set the tag.  Yes, it is based on the event of the other object, but that's not how you're using it.

Reply:
I feel it's more meaningful to call it event, which is why I chose the name. The tag is kind of an implementation detail. I don't really care though.

Reply:
the main reason is that event is an entirely different thing, so it's better to name it the actual thing you're using it for

Reply:
please follow the convention of Tim Sweeney's `spawn` function being called, which uses the name `tag` for that argument